website/content/commands/server-upgrade.mdx

```diff
@@ -52,6 +52,8 @@ manually installed runners will not be automatically upgraded.
 - `-ecs-region=<string>` - Configures which AWS region to install into.
 - `-ecs-cpu=<string>` - Configures the requested CPU amount for the Waypoint server task in ECS.
```

Comment:
Might want to mention this configures the waypoint ODR task cpu too, or make another config field for it. same with mem.

Reply:
It doesn't I think! I had the same question, I think cpu/mem for tasks are currently not configurable. Since you had the same feedback it's probably worth adding the fields.

Reply:
ODR cpu/mem is not configurable in this PR, which is consistent with other initial ODR implementations if I recall... we can add configuration in a follow up PR

Reply:
ODR memory and CPU are configurable as of c5c92919